source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,549 @@ Filters Builders
    :class: singlecol
 
 .. _filters-builders:
+
+Overview 
+--------
+In this guide, we show you how to use **builders** to specify
+**filters** for your queries.
+
+Builders are classes provided by the MongoDB Java driver that allow you
+to use the builder pattern to interact with your MongoDB instance. The
+builders provided by the MongoDB Java driver can improve the readability
+of your code.  See the :doc:`Builders
+</fundamentals/builders/>` page for a list of Builders available in the
```

Comment:
Nitpick: no change required, but I like to keep links (whether external links or directive-based ones) on the same line even if it goes beyond the 80 char width. In this case, I don't think it would even go beyond that width if you kept the Builders link on line 23.